sklearn/linear_model/ransac.py

```diff
@@ -111,6 +111,11 @@ class RANSACRegressor(BaseEstimator, MetaEstimatorMixin, RegressorMixin):
     max_trials : int, optional
         Maximum number of iterations for random sample selection.
 
+    max_skips : int, optional
+        Maximum number of iterations that can be skipped due to finding zero
+        inliers or invalid data defined by ``is_data_valid`` or invalid models
+        defined by ``is_model_valid``.
+
```

Comment:
I suppose nowadays we're meant to have `.. versionadded` annotations in docstrings.